packages/jest-mock/src/index.ts

```diff
@@ -1277,6 +1311,116 @@ export class ModuleMocker {
     return mock;
   }
 
+  mockProperty<
+    T extends object,
+    P extends PropertyLikeKeys<T>,
+    V extends Required<T>[P],
+  >(object: T, propertyKey: P, value: V): MockedProperty {
+    if (!object) {
+      throw new Error(
+        `mockProperty could not find an object on which to replace ${String(
+          propertyKey,
+        )}`,
+      );
+    }
+
+    if (!propertyKey) {
```

Comment:
Should this use a more strict comparison? Empty strings and `0` should be fine to mock.

Reply:
Fixed, thanks!